### DISCUSSION TITLE
Quick way to find out the mean of neighbors?

### DISCUSSION THREAD

**Discussion by jonathan-h-w** [pyg-team/pytorch_geometric#5505] [Q&A] [answered]
Given node features and edge_index, is there a quick way to find out the mean of neighbors for each node? Expecting the output tensor to be the same shape as node features.

**Comment by EdisonLeeeee** [comment, marked as answer]
```python
from torch_geometric.nn.aggr import MeanAggregation
row, col = edge_index
x_mean = MeanAggregation()(x[row], col)
```

**Reply by jonathan-h-w** [reply]
Thanks Edison! Should've looked into the docs more carefully before posting

**Reply by EdisonLeeeee** [reply]
It's okay. Glad to help you :)
